scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
Whichever way we print the timestamp, it should include timezone, so we can always convert it to UTC. So the question is, what timezone do we prefer? I think the local timezone of the machine running the scraper is a good default, and we should set it to Bucharest time, so it's easy for us to read. The log parser should read the timestamp - it's in standard ISO format - and store it however it wants.